### PR TITLE
Cleanup of index page and styles

### PR DESCRIPTION
**Pull request by ErikHellman**
**Description**
- Cleaned up the index page
- Cleaned up the global styles

**Purpose**
A first step to clean up the text and styles of the sampels.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,183 +9,168 @@
...
+            for instructions.</p>
 
+        <p class="warning"><strong>Warning:</strong> It is highly recommended to use headphones when testing these
+            samples, as it will otherwise risk loud audio feedback on your system.</p>
     </section>
 
     <section>
 
-      <h2 id="the-demos">The demos</h2>
+        <h2 id="getusermedia"><a href="https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getUserMedia">getUserMedia()</a>
```
Comment (samdutton) [inline index.html]: A few nitpicky suggestions...

• Title in bold followed by explanation in normal weight.
• Use a colon, not a hyphen.
• Use the imperative verb: 'Access media devices' not 'Accessing media devices'.

So, something like this:

**[getUserMedia](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getUserMedia)**: Access media devices

File: index.html [hunk 2]
```diff
@@ -9,183 +9,168 @@
...
+             - Accessing media devices</h2>
 
+        <li><a href="src/content/getusermedia/gum/">Basic getUserMedia demo</a></li>
```
Comment (samdutton) [inline index.html]: Wrap list items: `<ul> ... </ul>`.

File: index.html [hunk 3]
```diff
@@ -9,183 +9,168 @@
...
 
-      <h3 id="getusermedia">getUserMedia</h3>
+        <li><a href="src/content/getusermedia/canvas/">Use getUserMedia with canvas</a></li>
```
Comment (samdutton) [inline index.html]: Add more vertical space between list items, e.g. `margin: 0 0 0.5em 0;`.

File: index.html [hunk 4]
```diff
@@ -9,183 +9,168 @@
...
+        <li><a href="src/content/getusermedia/audio/">Audio-only getUserMedia() output to local audio element</a></li>
 
-      <p><a href="src/content/getusermedia/resolution/">Choose camera resolution</a></p>
+        <li><a href="src/content/getusermedia/volume/">Audio-only getUserMedia() displaying volume</a></li>
 
-      <p><a href="src/content/getusermedia/audio/">Audio-only getUserMedia() output to local audio element</a></p>
+        <li><a href="src/content/getusermedia/record/">Record stream</a></li>
 
-      <p><a href="src/content/getusermedia/volume/">Audio-only getUserMedia() displaying volume</a></p>
+        <h2 id="devices">Devices - Querying media devices</h2>
```
Comment (samdutton) [inline index.html]: Add a little more space between each group of links.

You could probably do this best by adding margin-bottom to each list, something like this: 
```
ul {
  margin: 0 0 0.5em 0;
}
```

**Review by guidou — APPROVED** [review]

**Pull request merged**